template/src/main/java/com/asyncapi/model/$$schema$$.java

```diff
@@ -1,44 +1,62 @@
 package com.asyncapi.model;
 
+import javax.validation.constraints.*;
+import javax.validation.Valid;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+{% if schema.description() or schema.examples() %}/**
+ * {{schema.description() | safe}}{% if schema.examples() %}
+ * Examples: {{schema.examples()}}{% endif %}
```

Comment:
This might introduce text with multiple lines. The new lines will not start with `*`.

Reply:
code will compile, but formatting is wrong, yeah. i'll fix this

Reply:
<img width="470" alt="Screen Shot 2020-04-19 at 12 45 09" src="https://user-images.githubusercontent.com/242119/79685786-a91aab00-823b-11ea-804f-682242ba7eb6.png">

The bug is still there :)

Reply:
You may want to get inspired by this: https://github.com/asyncapi/nodejs-template/blob/master/template/src/api/handlers/%24%24channel%24%24.js#L14.

Reply:
@fmvilas sorry, i do not see problems in your screenshot. 
Could you please provide example of schema that will produce a bug?

Reply:
Ohh, i see it now, after several experiments i found it. You meant that json objects from examples are rendered as [object Object] instead of some meaningful result.

Reply:
Hey @fmvilas please check new version.
I tested it with
```
    sentAt:
      type: string
      format: date-time
      description: Date and time when the message was sent.
      examples: [{key: 'value', inherit: {key1: 1, key2: 'c'}}, 'example',['array','string'], "try
      it
      multiline"]
```
and got:
```
    /**
     * Date and time when the message was sent.
     * Examples: {"key":"value","inherit":{"key1":1,"key2":"c"}}, example, ["array","string"], try it multiline
     */
```

For case of multi-line description like
```
      description: |
        This element represent data of one person.
        Also contains data about living place.
```
i got:
```
    /**
     * This element represent data of one person.
     * Also contains data about living place.
     * 
     */
```